.github/workflows/pythonapp.yml

```diff
@@ -17,19 +17,19 @@ jobs:
   build:
 
     runs-on: ${{ matrix.os }}
-    strategy: 
+    strategy:
       max-parallel: 6
-      matrix : 
+      matrix :
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: ['3.7', '3.8', '3.9', '3.10']
+        python-version: ['3.8', '3.9', '3.10']
```

Comment:
curious: do 3.11 and 3.12 work?

Reply:
No, some weird dependency installation failure on Windows with 3.12. See #253 (WIP).